cli/cmd/install.go

```diff
@@ -299,11 +308,20 @@ func (i *InstallCmdHandler) doInstallation(installParams installCmdParams) error
 
 func fetchContinuousDeliveryCharts(helmHelper helm.IHelper, chartRepoURL *string) ([]*chart.Chart, error) {
 	charts := []*chart.Chart{}
+	var serviceChart *chart.Chart
+	var err error
 	for _, service := range continuousDeliveryServices {
 		chartURL := getExecutionPlaneServiceChartRepoURL(chartRepoURL, service)
-		serviceChart, err := helmHelper.DownloadChart(chartURL)
-		if err != nil {
-			return nil, err
+		if goutils.IsValidURL(chartURL) {
```

Comment:
this code block is 100% the same code as in line `206+`. you can extract that into a separate func.